VERSION

```diff
@@ -1,4 +1,4 @@
-4.12.0+multicore
+4.12.0+effects
```

Comment:
Are there any downsides to naming it `4.12.0+domains+effects`? I'm worried this might give an impression to users that the branch has only effects and not domains.